test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -0,0 +1,267 @@
+- desc: Scheduling
```

Comment:
Let's rename to SchedulingBasic (to separate the basic tests case easily from others by Prefix matching)

Reply:
Done